### PR TITLE
Contrib test suite passing for all supported platforms.

### PR DESCRIPTION
**Pull request by AgarFu**
### Description

All supported platforms passing the contrib test suite with very basic integration tests.

### Contribution Check List

- [X] All tests pass.
- [X] New functionality includes testing.
- [X] New functionality has been documented in the README if applicable

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -14,8 +14,7 @@ install: echo "skip bundle install"
 
 branches:
   only:
-    - master
-    - kitchen_refactor
+    - /.*/
```
Comment (tas50) [inline .travis.yml]: If you open a WIP PR Travis will always test the PR. That way we avoid double testing on PRs with local branches, which takes forever to run with all these integration tests.

File: resources/extension.rb [hunk 2]
```diff
@@ -55,3 +57,9 @@ def extension_installed?
   query = "SELECT 'installed' FROM pg_extension WHERE extname = '#{extension}';"
   !(execute_sql(query, database) =~ /^installed$/).nil?
 end
+
+def check_extensions_support
+  query = 'SELECT version();'
+  version = execute_sql(query, database).split(' ')[1]
+  fail "PostgreSQL version #{version} does not support extensions. Minimmum required version: 9.1" if Gem::Version.new(version) < Gem::Version.new('9.1')
```
Comment (tas50) [inline resources/extension.rb]: Thanks Redhat 6... :(

File: test/cookbooks/test/recipes/contrib.rb [hunk 3]
```diff
@@ -2,6 +2,6 @@
 apt_update 'update' if platform_family?('debian')
 
 node.default['postgresql']['password']['postgres'] = 'iloverandompasswordsbutthiswilldo'
-node.default['postgresql']['contrib']['extensions'] = %w(plpgsql xml2)
+node.default['postgresql']['contrib']['extensions'] = %w(plpgsql xml2) unless platform_family?('rhel') && Gem::Version.new(node['platform_version']) < Gem::Version.new('7')
```
Comment (tas50) [inline test/cookbooks/test/recipes/contrib.rb]: It's probably simpler here to just do node['platform_version'].to_i instead of using Gem::Version

**Comment by tas50** [comment]
@AgarFu Any updates on this PR. It would be great to get some of those contrib fixes in.

**Comment by tas50** [comment]
Hit the wrong button there. Sorry

**Comment by AgarFu** [comment]
Fixed.

**Comment by tas50** [comment]
Looks good. Thanks

**Pull request merged**